index.js

```diff
@@ -12,6 +12,23 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
 
     if (defaultBranch && settingsModified) {
       return Settings.sync(context.github, context.repo())
+    } else if (!defaultBranch && settingsModified) {
+      let settingChangeCommits = payload.commits.filter(commit => { // get all commits that change settings.yml
+        return commit.added.includes(Settings.FILE_NAME) ||
+        commit.modified.includes(Settings.FILE_NAME)
+      })
+      settingChangeCommits.forEach(async function (commit, index, array) { // create status for each commit that changed settings.yml
+        try {
+          await context.github.repos.createStatus({
+            owner: payload.repository.owner.login,
+            repo: payload.repository.name,
+            sha: commit.id,
+            state: 'error'
```

Comment:
The status of this should only be `error` if the configuration file can't be parsed, or if it contains values that the GitHub API doesn't understand (which is going to be an interesting problem to solve).

Reply:
@bkeepers Could you please tell a bit more about the second part? I'm curious.